src/api/institutionApi.ts

```diff
@@ -0,0 +1,81 @@
+import Axios from 'axios';
+import { API_URL } from '../utils/constants';
+import {
+  Institution,
+  InstitutionSubUnit,
+  emptyInstitution,
+  emptyInstitutionSubUnit,
+  InstitutionPresentationModel,
+  emptyInstitutionName,
+} from '../types/institution.types';
+
+export enum InstituionApiPaths {
+  QUERY = '/institution',
+  GET = '/institution/unit/',
+}
+
+export const queryInstitution = async (searchTerm: string) => {
```

Comment:
vi har som regel brukt å kalle funksjoner som GETer noe for getInstitution

Reply:
Eirik: Dette er en query som gir flere institutsjoner som output. En get burde etter min mening gi et objekt som resultat.